pkgs/by-name/el/elm-land/package.nix

```diff
@@ -0,0 +1,75 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  elmPackages,
+  versionCheckHook,
+  writeShellScript,
+  nix-update,
+  elm2nix,
+}:
+
+buildNpmPackage rec {
+  pname = "elm-land";
+  version = "0.20.1";
+
+  src = fetchFromGitHub {
+    owner = "elm-land";
+    repo = "elm-land";
+    rev = "v${version}";
+    hash = "sha256-PFyiVTH2Cek377YZwaCmvDToQCaxWQvJrQkRhyNI2Wg=";
+  };
+
+  sourceRoot = "${src.name}/projects/cli";
+
+  npmDepsHash = "sha256-Bg16s0tqEaUT+BbFMKuEtx32rmbZLIILp8Ra/dQGmUg=";
+
+  postConfigure =
+    (elmPackages.fetchElmDeps {
+      elmPackages = import ./elm-srcs.nix;
+      elmVersion = elmPackages.elm.version;
+      registryDat = ./registry.dat;
+    })
+    + ''
+      # Use elm from nixpkgs
+      ln -sf ${lib.getExe elmPackages.elm} node_modules/.bin/elm
```

Comment:
Right this is exactly what I meant in https://github.com/NixOS/nixpkgs/pull/382339#issuecomment-2661018936 :+1: